tests/python/gaia-ui-tests/gaiatest/tests/functional/email/manifest.ini

```diff
@@ -4,6 +4,9 @@
 online = true
 smoketest = true
 
+[test_only_one_header_displayed.py]
```

Comment:
Fine by me. This test is already in the email/ directory, though, so I would think it's already pretty clear that this is an email test.